tests/testthat/setup-workflow-ref.R

```diff
@@ -80,16 +80,16 @@ CONFIG_DATA_MD5 <- "4ddb44da897c26681d892aa7be99f74b"
 CONFIG_MODEL_MD5 <- "6b930119c4224ba077091b47959b0604"
 
 # yaml md5 hashes
-MOD1_YAML_MD5 <- "ee5a30a015c4e09bc29334188ff28b58"
-MOD_LEVEL2_MD5 <- "1c18961a778fc6b71e3990b500f4bca5"
-ALL_MODS_YAML_MD5 <- c(MOD1_YAML_MD5, rep("22c7176b3dbbbf932091c82c2fd9f749", 2), "edd548af82d8556971c442f36a3907e0")
-RUN_LOG_YAML_MD5 <- c(MOD1_YAML_MD5, "62fc26bf3bf941c6bad352f23cc6c2ad", "ebd4861c9d0584b03a23edcfdda9a67e")
+MOD1_YAML_MD5 <- "6ccf206e167485b5adf29bc135197929"
```

Comment:
These were all changed because of this https://github.com/metrumresearchgroup/rbabylon/pull/267/files#r516199955

I was initially annoyed by having to change these for something that simple, but I think having `model_type` first is actually better, so this is the right move.